### ISSUE TITLE
Lambda Payload is not sent with version 2.8.22

### ISSUE THREAD

**Issue by bensigma** [aws/aws-sdk-php#801]
I realized today that with the new version 2.8.22 the payload is not sent along the request when calling invoke.

my call (works with 2.8.21 but not 2.8.22):
```php
$result = $this->lambda->invoke(array(
				// FunctionName is required
				'FunctionName' => 'foo',
				'InvocationType' => 'RequestResponse',
				'LogType' => 'None',
				'Payload' => '{"key":"value"}',
			)); 
```
Cheers Ben

**Comment by jeskew** [comment]
Thanks for reporting this! #802 should fix the issue you're seeing.

**Comment by jeskew** [comment]
#802 was included as part of 2.8.23. Please feel free to reopen if you have any questions or concerns.
